pshtt/cli.py

```diff
@@ -66,6 +66,8 @@ def main():
     if args['--json']:
         output = utils.json_for(results)
         if out_file is None:
+
+            utils.debug("Printing JSON...")
```

Comment:
We specifically omit debugging lines when outputting JSON or Markdown to STDOUT, so that downstream clients can pipe the output of `pshtt` directly into other processes that expect properly formatted JSON. Mind removing these new debugging output lines from the JSON and Markdown emitters?